### ISSUE TITLE
Fix License check (Library 1)

### ISSUE THREAD

**Issue by lukas-monet** [amosproj/amos2023ws03-gui-frame-diff#147]
## User story
1. As a user
2. I want to make sure that all licenses are fulfilled correctly
3. So that I can reuse the library without licensing problems

## Acceptance criteria
* license check for Library 1 is fixed
* all project licenses are written to the license report
* no license policy is violated

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
